client.lua

```diff
@@ -89,6 +94,10 @@ Citizen.CreateThread(function()
 
 			TriggerEvent('esx_teleportpads:hasExitedMarker', LastPad, LastAction)
 		end
+
+		if not HasAlreadyEnteredMarker then
```

Comment:
Will this really work?

Reply:
Yes

https://github.com/benzon/DRP_RECREATION/blob/af71c46e4b55e0a66f434a14abf718a6dc3c47a2/client/client.lua#L133

Doint the same in this resource, since HasAlreadyEnteredMarker is always set to false from the get go, it was better to tap in to that var, then creating a new one, and if you enter the marker it will be set to true and avoid the sleep function, it gives a tiny delay when you enter the ring, but it gives better performance in general.

Reply:
With out that it will have a idle of 0.03 it's about the same if you dont got the canSleep on DrawMarker, with the change the idle when out of range is 0.01 ms on my computer adleest.